### ISSUE TITLE
Outdated GHData

### ISSUE THREAD

**Issue by srobins259** [kalepeterson/InetSysDev-GH-Community-Health#32]
I just noticed that your repo still uses the GHData version before the configuration file change. This shouldn't change anything functionally, but I think they may have made some background changes between these versions. You may just want to consider trying to merge into the new version. I heard that it has a really nice front-end now that has a very well developed form!

**Comment by srobins259** [comment]
Actually, upon review, it looks like you do have it at least very close to up to date, but the front-end is returning a 404 error.

**Comment by kalepeterson** [comment]
Yes, merging the new ghdata design is going to be quite the chore it seems, so unfortunately this will remain an issue for a little while.

The front end is not running in our project.  We did not include directions on running it since we did not enhance it at all, we only added new REST endpoints.  We figured this out from the [developerstartup.md](https://github.com/kalepeterson/ghdata/blob/master/devloperstartup.md) file on the main GHData repo.  The last step is particularly relevant.

Hope this helps!
